### ISSUE TITLE
Need term for δ-aminolevulinate dehydratase activity low

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9662]
**Preferred term label:**
δ-aminolevulinate dehydratase activity low

**Synonyms**
ALAD; PBG synthase

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
I had this listed as HP:0034544 but there is no such term; it might have been an error on my part

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acute hepatic porphyria

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
I added this a month ago as
Decreased delta-aminolevulinate dehydratase activity
HP:0034544
It will become visible on the webpage/downloads with the April release

**Comment by MickeySegal** [comment]
I apparently neglected to add this to my list of terms we have added.

Thanks for clearing this up.
